code/modules/mob/living/simple_animal/abnormality/he/red_shoes.dm

```diff
@@ -0,0 +1,448 @@
+		living_pawn.cut_overlay(mutable_appearance('icons/effects/32x64.dmi', "panicked", -ABOVE_MOB_LAYER))
+		target.Assimilate(living_pawn)//breaches red shoes with target as the argument for user
+	controller.blackboard[BB_INSANE_CURRENT_ATTACK_TARGET] = null
+
+//*** Work Mechanics ***//
+/mob/living/simple_animal/hostile/abnormality/red_shoes/proc/Apply_Desire(mob/living/carbon/human/user)
+	user.apply_status_effect(STATUS_EFFECT_REDPOSSESS)//desire applied
+	playsound(src, 'sound/abnormalities/redshoes/RedShoes_Activate.ogg', 100, 1)
+	return
+
+/mob/living/simple_animal/hostile/abnormality/red_shoes/FailureEffect(mob/living/carbon/human/user, work_type, pe)
+	datum_reference.qliphoth_change(-1)
+	return
+
+/mob/living/simple_animal/hostile/abnormality/red_shoes/PostWorkEffect(mob/living/carbon/human/user, work_type, pe)
+	if(get_attribute_level(user, TEMPERANCE_ATTRIBUTE) < 60)
+		Apply_Desire(user)
+		user.adjustSanityLoss(500)
+		for(var/turf/T in range(5, src))
+			for(var/mob/living/carbon/human/H in T)
+				to_chat(H, "<span class='userdanger'>[user] ignores [p_they()] orders and continually glances at The Red Shoes. Now [p_theyre()] reaching out their hand to take the shoes.</span>")
+				return ..()
+
+//***Breach Mechanics***//
+/mob/living/simple_animal/hostile/abnormality/red_shoes/ZeroQliphoth(mob/living/carbon/human/user)//silent girl with extra steps
+	if(!(status_flags & GODMODE)) // If it's breaching right now
+		return
+	if(possessee)//If the first check fails
+		return
+	SLEEP_CHECK_DEATH(30)
+	if(LAZYLEN(GLOB.player_list) < 3)//solo breach if there aren't many players
+		BreachEffect()
+		return
+	numbermarked = (1 + round(LAZYLEN(GLOB.player_list) / 6))
+	var/list/potentialmarked = list()//mark code from cherry
+	var/list/marked = list()
+	for(var/mob/living/carbon/human/L in GLOB.player_list)
+		if(L.stat >= HARD_CRIT || L.sanity_lost || z != L.z) // Dead or in hard crit, insane, or on a different Z level.
+			continue
+		potentialmarked += L
+	for(var/i=numbermarked, i>=1, i--)
+		var/mob/living/Y = pick(potentialmarked)
+		if(faction_check_mob(Y, FALSE) || Y.z != z || Y.stat == DEAD)
+			continue
+		if(Y in marked)
+			continue
+		marked+=Y
+		Apply_Desire(Y)
+	datum_reference.qliphoth_change(2)
+	return
+
+/mob/living/simple_animal/hostile/abnormality/red_shoes/proc/Assimilate(mob/living/carbon/user)
+	if(!(status_flags & GODMODE))
+		return
+	if(possessee)
+		return
+	var/list/search_area = user.contents//get items from the possessee
+	possessee = user
+	var/mob/living/carbon/human/H = user
+	if(ishuman(H) && (H.sanity_lost))//also assimilates fortitude panics
+		for(var/obj/item/clothing/suit/armor/ego_gear/EQ in search_area)//copies all resistances from worn E.G.O
+			for(var/damtype in damage_coeff)
+				if(damtype == BRUTE)
+					continue
+				damage_coeff[damtype] -= (EQ.armor[damtype] / 100)
+			break//check only the first
```

Comment:
You could probably check the actual worn armor slot, because you might grab something they're otherwise holding.

Reply:
Again, check O_CLOTHING